spec/lib/tasks/user_spec.rb

```diff
@@ -12,15 +12,19 @@
 
   let(:user) { users(:affiliate_manager_with_no_affiliates) }
   let(:not_active_user) { users(:not_active_user) }
+  let(:task_name) { 'usasearch:user:update_approval_status' }
```

Comment:
I would move this back to its original location, within the `describe` block that it applies to:
```ruby
describe 'usasearch:user:update_approval_status' do
  let(:task_name) { 'usasearch:user:update_approval_status' }
```

Reply:
ah miss understood your slack message thought you wanted all the let defs together. Makes sense.